src/site/stages/build/plugins/add-id-to-subheadings.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Add unique ID to H2s and H3s that aren't in WYSIWYG or accordion buttons
+ */
+
+const cheerio = require('cheerio');
+
+const usedHeaders = [];
+
+let g = 1;
```

Comment:
Could we name this `currentId` or something? I'm not sure what `g` means, exactly.